processing/src/main/java/org/apache/druid/query/aggregation/last/StringLastVectorAggregator.java

```diff
@@ -64,8 +64,9 @@ public void aggregate(ByteBuffer buf, int position, int startRow, int endRow)
     if (timeSelector == null) {
       return;
     }
-    long[] times = timeSelector.getLongVector();
-    Object[] objectsWhichMightBeStrings = valueSelector.getObjectVector();
+    final long[] times = timeSelector.getLongVector();
+    final boolean[] nullTimeVector = timeSelector.getNullVector();
```

Comment:
is length of nullTimeVector guaranteed to be aligned with startRow and endRow?

Reply:
Yes the part of the value received within the startRow and endRow will also be the length of the null vector if present